privly.safariextension/scripts/background_scripts/first_run.js

```diff
@@ -47,6 +47,7 @@ var firstRun = {
       ls.setItem("glyph_cells", glyph_cells);
     }
 
+    /* istanbul ignore if */
     if (typeof safari !== "undefined" && safari.application !== undefined) {
```

Comment:
You should probably define a helper function, since this "if" statement will be repeated elsewhere.